daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -494,8 +494,10 @@ func (d *Driver) Remove(id string) error {
 	if err := os.RemoveAll(dir); err != nil && !os.IsNotExist(err) {
 		return err
 	}
-	if err := subvolRescanQuota(d.home); err != nil {
-		return err
+	if quotaEnabled {
```

Comment:
I was looking at the code, and a bit in doubt. The `quotaEnabled` boolean
is set _either_ if [`--storage-opt size=btrfs.min_space`](https://github.com/docker/docker/blob/v1.12.5/docs/reference/commandline/dockerd.md#btrfs-options) is set globally (on the daemon), or if a `size` is passed when creating a container (see [set storage driver options per container](https://github.com/docker/docker/blob/v1.12.5/docs/reference/commandline/run.md#set-storage-driver-options-per-container))

I'm not too familiar with this part of the code; is the [`Create()` method](https://github.com/docker/docker/blob/v1.12.5/daemon/graphdriver/btrfs/btrfs.go#L384) also called when _starting_ an existing container, or only the first time its filesystem is created?

Otherwise, this may not work, because the `quotaEnabled` variable is only set here; https://github.com/docker/docker/blob/v1.12.5/daemon/graphdriver/btrfs/btrfs.go#L461-L466
(in situations where `btrfs.min_space` is not set on the daemon)

So, maybe this is the right change, but I'm just writing down my thoughts, in case it's useful.

ping @cpuguy83 you probably know